metadata.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "ghoneycutt-hosts",
-  "version": "2.2.3",
+  "version": "2.2.4",
```

Comment:
I created patch-1 branch from 2.2.3 version and after your last week pull request merge I rebased patch-1 branch against last puppet-modules-host version (v2.2.4) before proceding with changes. I don't know why metadata.json is marked as modified.